communication/requesting-survey.md

```diff
@@ -0,0 +1,102 @@
+* Are there obvious missing options to some questions?
+* Is the preface clear and complete?
+* Does the survey comply with the Kubernetes CoC and privacy practices?
+
+Once you feel like the survey is as good as it's likely to be, we can publish it.
+
+### 3. Request Survey Publication
+
+By default, we use a SurveyMonkey account funded by the CNCF to publish surveys.  This is because SurveyMonkey offers a fairly rich set of survey options, and more importantly is accessible in China.  We can also publish surveys using the Kubernetes Google Drive account, if surveying Chinese developers or contributors is not a consideration.  GDrive makes sharing the survey results easier.
+
+In either case, it will take a few days for a ContribEx volunteer to copy the questions and options into the survey tool, and then it can be ready to announce on your schedule.  Just comment on the issue giving your requested publication schedule, and mark it LGTM.
+
+### 4. Contributor Communications Promotion (optional)
+
+If you want your survey to reach a large number of contributors, developers, or users, then the Contributor Marketing subproject can help.  Tag `/area contributor-comms` on the issue, and request help from that group.  They will ask you some questions about promoting the survey and assist you in getting the word out, via the Kubernetes social medial accounts, mailing lists, and blog, depending on the target audience and scope.
+
+### 5. Collect Survey Results
+
+Your survey can be open anywhere from a few days to a few months, depending on your needs.  In addition to any efforts Contributor-Comms may be making on your behalf, during this period your group should make sure to remind folks several times that the survey is open.  
+
+On request (via the Github issue or Slack messages) ContribEx volunteers can supply intermediate information, such as the number of surveys completed or the current results data.
+
+### 6. Publish Survey Results
+
+When the survey is complete, the designated recipients for the raw data will receive a data dump in the form of a spreadsheet or CSV file, accompanied by copies of the simple graphs created by the survey system.  As this data may contain personally identifying information, the recipients are required to be a small group who will treat the data with care.
+
+Depending on the availability of some ContribEx members, you may also be able to get more advanced visualizations and/or data analysis assistance.  Request this in your issue and we'll follow-up to help figure out exactly what you need.
+
+## Tips and Notes Around Surveys
+
+Please note that we are subject to the GDPR and other privacy regulations.  As such, your survey will need to explain who will have access to the data and what it's for, and you need to adhere to those restrictions.  Among other things, this means that you can never share raw survey data on a public mailing list or or GitHub repo (summary data should be fine).
```

Comment:
Suggestion: Be explicit -- "... summary data with no identifyable information should be fine"

Example: If there's only one Kubernetes Contributor from Ann Arbor, and a summary or presentation shows that person from Ann Arbor, that's arguably identifyable but still a "summary". 

Better to be safe /shrug